`swag` binary should be installed before running `make swag`

make swag 실행 시 swag 실행 파일이 없으면 오류가 발생합니다.
```
$ make swag
cd src/ && make swag
make[1]: Entering directory '/home/sykim/workspace/cb-tumblebug/src'
~/go/bin/swag i -o ./api/rest/docs
/bin/sh: 1: /home/sykim/go/bin/swag: not found
make[1]: *** [Makefile:26: swag] Error 127
make[1]: Leaving directory '/home/sykim/workspace/cb-tumblebug/src'
make: *** [Makefile:20: swag] Error 2
```
간단하게 설치 방법 명시가 필요해 보입니다.

`go install github.com/swaggo/swag/cmd/swag@latest`

해당 내용이 간단히 언급되어 있어 직접 PR을 작성하지 않고 의견만 드립니다.

https://github.com/cloud-barista/cb-tumblebug/blob/ca1f34576bbab3b14ddb48092bd77e8a9904bdf6/README.md?plain=1#L559

--- a/src/Makefile
+++ b/src/Makefile
@@ -23,5 +23,11 @@ clean:
 	rm -v cb-tumblebug cb-tumblebug-arm
 
 swag swagger:
+	@echo ""
+	@echo "This commend requires swag binary."
+	@echo "If you don't have it, please install swag first with the following commend"
+	@echo "- go install github.com/swaggo/swag/cmd/swag@latest"
+	@echo ""
+
 	~/go/bin/swag i -o ./api/rest/docs
 	../scripts/misc/convert-swagger-version.sh

